tutorial/datastructures.po

```diff
@@ -122,12 +123,12 @@ msgid ""
 "Sort the items of the list in place (the arguments can be used for sort "
 "customization, see :func:`sorted` for their explanation)."
 msgstr ""
-"Classe les éléments sur place (les arguments peuvent personnaliser le "
-"classement, voir :func:`sorted` pour leur explication)."
+"Tri les éléments (les arguments peuvent personnaliser le tri, voir :func:"
```

Comment:
Une traduction pour `in place` disparaît, et je la trouve importante pour bien contraster entre `sorted` et `.sort`: `sorted` renvoie une nouvelle liste, triée, et `.sort` trie "sur place".

Reply:
OK, je ne comprenais pas ce que le "in place" signifiait. Je trouve que "Tri les éléments directement dans la liste" ou "Tri les éléments en modifiant la liste" est plus clair. Si ça te va, j'applique la même correction au Reverse.

Reply:
J'ai une préférence pour ton "Tri les éléments directement dans la liste", mais j'ai un doute sur "directement", on peut peut être l'enlever. Je veux dire par là que je ne voit pas trop ce qu'est un "Trier indirectement". Le contriare de "in place" est "renvoie une nouvelle liste triée", pas de notion d'indirectement ici.